### PR TITLE
Bump core version

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: sdk/core/Azure.Core/tests/ArrayBackedPropertyBagTests.cs [hunk 1]
```diff
@@ -167,14 +167,8 @@ public void Dispose(int count)
...
-#if DEBUG
-            Assert.Throws<ObjectDisposedException>(() => { _ = target.Count; });
-            Assert.Throws<ObjectDisposedException>(() => { _ = target.IsEmpty; });
-            for (var key = 0; key < count; key++)
-            {
-                Assert.Throws<ObjectDisposedException>(() => { _ = target.TryGetValue(key, out _); });
-            }
-#else
+
+#if !DEBUG
```
Comment (christothes) [inline sdk/core/Azure.Core/tests/ArrayBackedPropertyBagTests.cs:171]: Is this related to the version bump?

Comment (AlexanderSher) [reply sdk/core/Azure.Core/tests/ArrayBackedPropertyBagTests.cs:171]: Yes. Test fails in combination "Azure.Core - Release" + "Azure.Core.Test - Debug" which becomes possible when Azure.Core is referenced as package

**Review by AlexanderSher — APPROVED** [review]

**Review by jsquire — APPROVED** [review]
